assert/assertions.go

```diff
@@ -144,19 +180,20 @@ func Fail(t TestingT, failureMessage string, msgAndArgs ...interface{}) bool {
 
 	message := messageFromMsgAndArgs(msgAndArgs...)
 
+	locationInfo := strings.Join(CallerInfo(), "\n\r\t\t\t")
 	if len(message) > 0 {
 		t.Errorf("\r%s\r\tLocation:\t%s\n"+
```

Comment:
It's no longer a Location, but an error trade.

Maybe we should change the name?